client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx

```diff
@@ -34,7 +34,7 @@ const ViewSeasonInfo = (props: ViewSeasonInfoProps) => {
 
   return (
     <header className='rounded-xl bg-white p-7 text-lg'>
```

Comment:
Could you add `shadow-xl` to this as well to match look of events page

Reply:
Got it!